util.go

```diff
@@ -167,35 +173,36 @@ func (s byLength) Less(i, j int) bool {
 
 // Prepares strings by splitting by caps, spaces, dashes, and underscore
 func split(str string) []string {
-	repl := strings.NewReplacer(
-		"@", "At ",
-		"&", "And ",
-		"|", "Pipe ",
-		"$", "Dollar ",
-		"!", "Bang ",
-		"-", " ",
-		"_", " ",
-	)
-
-	rex1 := regexp.MustCompile(`(\p{Lu})`)
-	rex2 := regexp.MustCompile(`(\pL|\pM|\pN|\p{Pc})+`)
+	// check if consecutive single char things make up an initialism
+	once.Do(func() {
```

Comment:
Does it make a difference if you move this once out too and put it in an init() hook?